roles/development/templates/default.conf

```diff
@@ -0,0 +1,85 @@
+server {
+    listen       80;
+    server_name  {{ nginx_server_name }};
+
+    root /projects/{{ nginx_server_name }}/website/;
```

Comment:
@wakawakaxin 
I think it is better, that we use the project name for the path. And the server name is combined by the project name and the domain name